lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
```

Comment:
To make this consistent with the main project, should be make this `lib/collections/schemas`?